modules/cluster-logging-deploy-cli.adoc

```diff
@@ -55,10 +55,10 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging <1>
 spec:
-  targetNamespaces:
-  - openshift-logging <1>
+  targetNamespaces: [ ] <2>
 ----
-<1> You must specify the `openshift-logging` namespace for logging versions 5.7 and older. For logging 5.8 and later versions, you can use any namespace.
+<1> Ensure that the `namespace` field is set to `openshift-logging`.
+<2> For logging versions 5.7 and older, set `targetNamespaces` to `openshift-logging`. For logging versions 5.8 and later, the default installation mode is all namespaces on the cluster, as shown in the example, or specify a list of namespaces.
```

Comment:
```suggestion
<2> For logging versions 5.7 and earlier, set `targetNamespaces` to `openshift-logging`. For logging versions 5.8 and later, the default installation mode is all namespaces on the cluster, or specify a list of namespaces separated by a comma.
```
I dont think "as specified in the example is needed here because it is a callout. Also, I added seperated by a comma. I am not sure if that is accurate or not when you list out namespaces, but I think that is how I have seen it before. I think it would add some clarification. 

s/older/earlier based on ibm style.
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#:~:text=your%20own%20installation.-,earlier,-adverb

Reply:
> <2> For logging versions 5.7 and earlier, set `targetNamespaces` to `openshift-logging`. For logging versions 5.8 and later, the default installation mode is all namespaces on the cluster, or specify a list of namespaces separated by a comma.

Seems ok to me.

Cc @r2d2rnd @anpingli

Reply:
@r2d2rnd @anpingli Could you please confirm if we can use "separated by a comma" as suggested [here](https://github.com/openshift/openshift-docs/pull/75652#discussion_r1666720762)? Thank you!

Reply:
LGTM

Reply:
@opayne1 We have QE approval here for the update and so it seems we can use the "separated by a comma". Please help with merge of this PR.